extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -1,35 +1,152 @@
 define('wikia.articleVideo.featuredVideo.tracking', [], function () {
+	var state = getDefaultState();
+	var gaCategory = 'featured-video';
+	var playerInstance;
+	var wasAlreadyUnmuted = false;
+	var depth = 0;
+
 	function getDefaultState() {
 		return {
-			wasAlreadyUnmuted: false
+			wasStartTracked: false,
+			progress: {
+				durationTracked: 0,
+				percentTracked: 0
+			}
 		}
 	}
 
 	function track(gaData) {
-		console.log(gaData);
+		if (!gaData.label) {
+			throw new Error('No tracking label provided');
+		}
+
+		var finalGAData = {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'finalGAData' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "finalGAData" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)